requirements.txt

```diff
@@ -1,26 +1,33 @@
-cffi
+boto==2.38.0
+celery==3.1.19
+cffi==1.3.1
+cryptography==1.1.1
 Django==1.9
 django-appconf==1.0.1
-django-autofixture
-django-compressor
+django-autofixture==0.10.1
+django-compressor==2.0
 django-crispy-forms==1.5.2
 dj-database-url==0.3.0
 django-oauth-toolkit==0.9.0
 djangorestframework==3.3.1
+django-redis-sessions==0.5.0
+dj-redis-url==0.1.4
 dj-static==0.0.6
 django-toolbelt==0.0.1
-dropbox
-google-api-python-client
-gunicorn==19.4.5
-hashids
-httpie
+django-websocket-redis==0.4.6
+dropbox==3.23
+gevent==1.0.2
+google-api-python-client==1.4.1
+hashids==1.1.0
+hiredis==0.2.0
```

Comment:
Yes for websocket lib as it is a wrapper layer over Redis to handle long live connections.